spec/parser/option_spec.rb

```diff
@@ -105,11 +105,11 @@ def option(name, options = {})
 
     describe "with key as an array" do
       it "sets the first items in the array to the name" do
-        expect(parse([:foo, :bar, :baz], true).name).to eq("foo")
+        expect(parse([:foo, :b, "--bar"], true).name).to eq("foo")
```

Comment:
As far as I can tell, these two tests were invalid.

Specifying a multicharacter alias without including leading dashes leads to a single dash being prepended (rather than two), which is invalid, as there is no way to invoke it.

By example:

Specified Alias|Final Alias
-|-
`"-b"`, `:"-b"`, `"b"`, `:b`,|`-b`
`"--bar"`, `:"--bar"`|`--bar`
`"bar"`, `:bar`|`-bar` (invalid)

Therefore, I updated these tests to use legitimate aliases, which then make sense with normalization.

Specifying invalid aliases is still allowed, it just now will generate a single dash in the usage documentation (which reflects what is actually going on).

Another PR could address forbidding invalid aliases (as they do not work, and would be misleading) in isolation.